### DISCUSSION TITLE
How to change hotkeys?

### DISCUSSION THREAD

**Discussion by Ankras78** [alefragnani/vscode-bookmarks#414] [Q&A] [answered]
Hi, help pls:
1. How can I reassign for example "Ctrl+Alt+K" or  "Ctrl+Alt+L" to another key on keyboard ???
2. How can I reassign looong inputing in Command Palette "Bookmarks: Clear" to some key on keyboard ???
Thanks!

**Comment by alefragnani** [comment, marked as answer]
Hi @Ankras78,

VS Code supports customization of every keyboard shortcuts and commands. 

Just follow their documentation https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-shortcuts-editor. You can use a UI or update the `JSON` file directly.

Hope this helps

**Comment by Ankras78** [comment]
BIG THANKS!
